env/index.js

```diff
@@ -1,7 +1,7 @@
 const dotenv = require('dotenv');
 const fs = require('fs');
 const path = require('path');
-const ROOT_DIR = path.resolve(__dirname, '..', '..');
+const ROOT_DIR = path.resolve(__dirname, '..');
```

Comment:
is there expected to be a second `testenv' file in the 'test' directory? Or is this the new location? Our convention is that it is always in the repo root.

Reply:
n/m I see this file was moved to env.